mlcolvar/core/transform/descriptors/multiple_descriptors.py

```diff
@@ -0,0 +1,89 @@
+import torch
+
+__all__ = ["MultipleDescriptors"]
+
+class MultipleDescriptors(torch.nn.Module):
+    """Wrapper class to combine multiple descriptor transform objects acting on the same set of atomic positions in a single module"""
```

Comment:
why should they act on the same atoms? e.g. a single torsional angle cannot be combined with pairwise distances on others?

Reply:
To have a single object the forward must be common, so does be the input. 
If you want to combine different descriptors you take as input all the positions and then you use the indexing within each single class to use only the atoms you need (as for torsional angle) or return only the output you want (as for the distances)